python/lsst/obs/hsc/makeTransmissionCurves.py

```diff
@@ -36,20 +35,23 @@
 
 HSC_BEGIN = "2012-12-18"  # initial date for curves valid for entire lifetime of HSC
 
+OLD_FILTER_NAME_MAP = {b: f"HSC-{b.upper()}" for b in "grizy"}
+OLD_FILTER_NAME_MAP.update(r2="HSC-R2", i2="HSC-I2")
```

Comment:
Should we have a comment above this saying something like "this is to map physical filter names to the deprecated afw names", for future readers?